### PR TITLE
Warm welcome!

### PR DESCRIPTION
**Pull request by jandamm**
Refine welcome screen

### CODE REVIEW

**Comment by jandamm** [comment]
Hey Katha, please check!

File: XcodePRTest/ContentView.swift [hunk 1]
```diff
@@ -10,10 +10,10 @@ import SwiftUI
 struct ContentView: View {
     var body: some View {
         VStack {
-            Image(systemName: "globe")
+            Image(systemName: "person")
```
Comment (KatharinaGopp) [inline XcodePRTest/ContentView.swift:13]: please use "person.circle"

Comment (jandamm) [reply XcodePRTest/ContentView.swift:13]: Done.

**Review by KatharinaGopp — APPROVED** [review]
'APPROVE' performed by Xcode.

**Pull request merged**